test/testdata/fixtures/single-dc/k8ssandra.yaml

```diff
@@ -4,13 +4,13 @@ metadata:
   name: test
 spec:
   cassandra:
-    serverVersion: 3.11.11
+    serverVersion: 4.0.6
     clusterName: "Weird Cluster Name"
     datacenters:
       - metadata:
           name: dc1
         k8sContext: kind-k8ssandra-0
-        size: 1
+        size: 3
```

Comment:
Why do we need 3 nodes now?

Reply:
yeah, good question 🤔 
Sadly I saw the comments after the approval and merged right away.
We can revert these two changes if we see this test getting more flaky.